scripts/mail_resp.py

```diff
@@ -31,6 +32,7 @@
         u'Nemanja Rilakovic':'nemanja.rilakovic@scilifelab.se',
         u'Mattias Oskarsson':'mattias.oskarsson@scilifelab.se',
         u'Par Lundin':'par.lundin@scilifelab.se',
+        u'Phil Ewels':'phil.ewels@scilifelab.se',
         u'Remi-Andre Olsen':'remi-andre.olsen@scilifelab.se',
         u'Senthilkumar Paneerselvam':'senthilkumar.paneerselvam@scilifelab.se',
```

Comment:
My email address is "senthilkumar.panneerselvam@scilifelab.se" with 'nn' :stuck_out_tongue_closed_eyes: